### PR TITLE
Master watched

### PR DESCRIPTION
**Pull request by joeyfreeman**
-Added watched episodes functionality
-Changed AccountManager::getCurrentAccount() to retrieve an updated User from the DatabaseManager
-Modified DatabaseManager::getListOfWatchedTvShowNamesForUser() to only list a particular TV show once

### CODE REVIEW

**Review by haleyminshell — APPROVED** [review]
Looks good. Passed all unit tests and tested application and now it works with adding shows to watched list. Nice job!

File: PlotTalk/Src/accountmanager.cpp [hunk 1]
```diff
@@ -66,6 +66,7 @@ bool AccountManager::createAccount(User user)
 
 User &AccountManager::getCurrentAccount()
 {//retuns the account information of the account held in the program
+    referenceTodatabaseUser = DatabaseManagerSingleton::Instance().inspectUser(referenceTodatabaseUser.username);
```
Comment (allanhaywood) [inline PlotTalk/Src/accountmanager.cpp]: Should we change the name since it isn't a reference anymore?

**Review by allanhaywood — APPROVED** [review]
Minor feedback. Approving.

File: PlotTalk/Src/accountmanager.cpp (hunk 1 above)
Comment (joeyfreeman) [reply PlotTalk/Src/accountmanager.cpp]: Done

**Pull request merged**